monty-31/src/x86_64_avx2/packing.rs

```diff
@@ -4,6 +4,7 @@ use core::mem::transmute;
 use core::ops::{Add, AddAssign, Div, Mul, MulAssign, Neg, Sub, SubAssign};
 
 use p3_field::{AbstractField, Field, PackedField, PackedFieldPow2, PackedValue};
+use p3_util::convert_vec;
 use rand::distributions::{Distribution, Standard};
 use rand::Rng;
```

Comment:
I think you also need to add: use alloc::vec::Vec;

Reply:
Good catch, fixed